workspaces/ui-v2/src/optic-components/pages/diffs/ReviewEndpointDiffPage/ReviewEndpointDiffPage.tsx

```diff
@@ -71,29 +74,66 @@ export const ReviewEndpointDiffPage: FC<ReviewEndpointDiffPageProps> = ({
   });
   const endpointId = getEndpointId({ method, pathId });
 
-  const allDiffsHandled = shapeDiffs.every(
-    (shapeDiff) =>
-      shapeDiff?.diffDescription?.diffHash &&
-      isDiffHandled(shapeDiff.diffDescription.diffHash)
-  );
-  const [currentIndex, setCurrentIndex] = useState(0);
+  const getNextIncompleteDiff = (recentlyCompletedDiff?: string): number => {
+    for (let i = 0; i < shapeDiffs.length; i++) {
+      const shapeDiff = shapeDiffs[i];
+      if (!shapeDiff.diffDescription) {
```

Comment:
There's a bunch of things we can do, since it appears that this is always _set_ and never undefined (at least when it reaches this block of code) but for now, we'll just deal with the implicit checks - it looks like there's some usages of this where it's undefined (in which case we should probably have different types)